docker/tomcat/Dockerfile

```diff
@@ -5,7 +5,7 @@
 
 # Use the OpenJDK image
 # This builds on debian jessie
-FROM java:openjdk-8u66
```

Comment:
Currently on the testing we're using an old image that doesn't use openjdk but actually uses our custom java image.